addons/crm/models/crm_team.py

```diff
@@ -95,7 +95,15 @@ def _onchange_use_leads_opportunities(self):
     def write(self, vals):
         result = super(Team, self).write(vals)
         if 'use_leads' in vals or 'use_opportunities' in vals:
-            self.filtered(lambda team: not team.use_leads and not team.use_opportunities).alias_name = False
+            teams_without_alias = self.filtered(lambda team: not team.use_leads and not team.use_opportunities)
```

Comment:
Couldn't you simply call _alias_get_creation_values ?

Reply:
Well we only want to update `alias_defaults`, while `_alias_get_creation_values ` tries to update `alias_model_id ` which is not in `ALIAS_WRITEABLE_FIELDS` (and probably that's why one test was failing - [see this build](https://runbot.odoo.com/runbot/build/4552075))

Reply:
Right, good catch.

Maybe we could call _alias_get_creation_values and then get alias_defaults from the returned dict. That way we define logic only once. We will compute a bit more in dict than necessary but I don't think this will have a big impact (writing on teams is not a performance constrained flow).